appveyor.yml

```diff
@@ -80,18 +80,18 @@ init:
   - cmd: if /I %platform%==win32 7z -y x libtango_%cpptango_version%_%MSVCVERSION%_x86.zip -oC:\projects\libtango\
   - cmd: if /I %platform%==x64 7z -y x libtango_%cpptango_version%_%MSVCVERSION%_x64.zip -oC:\projects\libtango\
 
+  # TangoTest
+  - cmd: cd "C:\projects\"
+  - cmd: md tango_test
+  - cmd: cd "C:\projects\tango_test"
+  - appveyor DownloadFile https://github.com/spirit1317/tango-test-exe/raw/main/TangoTest.exe
```

Comment:
Not ideal, but we can use this until there is a better solution.

In the readme on that repo, could you add the exact commands you used to build the executable, please?